corehq/apps/geospatial/templates/map_visualization.html

```diff
@@ -102,6 +102,19 @@
         Previous disbursement was cleared.
     {% endblocktrans %}
 </div>
+<div id="disbursement-params" class="alert alert-info" data-bind="visible: showParams()">
+  <h4>Algorithm parameters</h4>
```

Comment:
I wonder if you want to call this "Disbursement Parameters"

Also, good to put this up for translation.
`trans 'Algorithm Parameters'`